hs/Server.hs

```diff
@@ -58,6 +63,9 @@ runServer = do
               dir "depts" $ liftIO deptList,
               dir "timesearch" searchResponse,
               dir "calendar" $ lookCookieValue "selected-lectures" >>= calendarResponse,
+              -- dir "draw-json?gid=num" $ look "gid" >>= drawJsonResponse,
```

Comment:
@R0ss it's perfectly acceptable to make a pull request just to get feedback, even if your work is incomplete. However, you generally want feedback on your actual code, and seeing changes which are just comments distracts from that.

@christinem this is a fine idea in general, but not suitable here because Ross' work should be able to be merged in incrementally.